confidant/routes/v1.py

```diff
@@ -102,13 +106,21 @@ def get_service(id):
     if authnz.user_is_user_type('service'):
         if not authnz.user_is_service(id):
             logging.warning('Authz failed for service {0}.'.format(id))
-            msg = 'Authenticated user is not authorized.'
+            msg = 'Service is not authorized.'
             return jsonify({'error': msg}), 401
     else:
+        logged_in_user = authnz.get_logged_in_user()
+        if not acl_module_check('get_service',
+                                metadata_only=metadata_only,
```

Comment:
I think we should check for allowed permissions here. So for instance, we'd pass in a list maybe? `['view', 'metadata']`

Reply:
Just updated along with adding the `acl_module_check` to additional endpoints. 

Lemme know if this is what ya had in mind.  
```
metadata_only = request.args.get('metadata_only', default=False, type=bool)
...
acl_actions = ['metadata']
if not metadata_only:
    acl_actions.append('get')
```
In the current logic, users default will be able to view full credentials unless `metadata_only` explicitly set to `True`